activiti-cloud-runtime-bundle-service/activiti-cloud-starter-runtime-bundle/src/test/java/org/activiti/cloud/starter/tests/runtime/MQServiceTaskIT.java

```diff
@@ -125,4 +137,91 @@ private void replayServiceTask(IntegrationContext integrationContext) {
         identityTokenProducer.withTestUser(keycloakTestUser);
         assertThat(responseEntity.getStatusCode()).isEqualTo(HttpStatus.OK);
     }
+
+    @Test
+    void shouldRemoveInboundAndOutBoundVarsFromIntegrationEventsForEphemeralVariables() {
+        auditConsumer.clear();
+        processInstanceRestTemplate.startProcess(
+            ProcessPayloadBuilder.start().withProcessDefinitionKey("ProcessWithRestConnectorEphemeralVars").build()
+        );
+        await()
+            .untilAsserted(() -> {
+                List<CloudIntegrationRequestedEvent> integrationRequestedEvents = auditConsumer.getAllReceivedEvents(
+                    CloudIntegrationRequestedEvent.class
+                );
+
+                IntegrationContextImpl integrationRequestEntity = (IntegrationContextImpl) (
```

Comment:
Unnecessary cast. You can use the interface everywhere as the three methods `hasEphemeralVariables`, `getInBoundVariables`, `getOutBoundVariables` are available directly from the interface